src/combinatorics/simple_graph/basic.lean

```diff
@@ -1114,6 +1161,15 @@ map_adj_iff f
 lemma card_eq_of_iso [fintype V] [fintype W] (f : G ≃g G') : fintype.card V = fintype.card W :=
 by convert (fintype.of_equiv_card f.to_equiv).symm
 
+/-- Equivalences of types induce isomorphisms of complete graphs on those types. -/
```

Comment:
Is there a version of this saying that if alpha embeds in beta, then the complete graphs of such also embed? Would it be useful to define this in terms of that?

Reply:
I see - in that case can we have the lemma that the (forward) graph embedding that comes from this equivalence agrees with the existing definition?

Reply:
Added